app/viewmodels/SummaryCardHelper.scala

```diff
@@ -289,6 +290,13 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
           link("#", "summaryCard.appeal.readMessageReinstated")
         )
       }
+      case AppealStatusEnum.Tribunal_Rejected => {
```

Comment:
I don't see a difference between this scenario and the Accepted | Rejected scenarios.
Couldn't you just add `| AppealStatusEnum.Tribunal_Rejected` to line 279? and remove this block?

Reply:
done